src/app/zap-templates/zcl/data-model/chip/tls-certificate-management-cluster.xml

```diff
@@ -0,0 +1,142 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2024 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<!--
+XML generated by Alchemy; DO NOT EDIT.
+Source: src/tls/TLSCertificateManagement.adoc
```

Comment:
I cannot find the `tls` directory in https://github.com/CHIP-Specifications/connectedhomeip-spec/tree/master/src

Should we wait for the spec to at least have a provisional item for this? What is the development order here? I am concerned that we may merge things that will become different from the spec.

Reply:
You have to look at the camera branch, linked from PR: https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/10004